documentation/docs/getting-started/environment-setup.md

```diff
@@ -1,85 +0,0 @@
----
```

Comment:
What's the reason for deleting this page? Do we not think it's necessary?

Reply:
I deleted it because it's pretty uber-specific